src/main/java/teammates/storage/api/FeedbackQuestionsDb.java

```diff
@@ -150,13 +150,33 @@ public FeedbackQuestionAttributes getFeedbackQuestion(
     
     /**
      * Updates the feedback question identified by `{@code newAttributes.getId()} 
+     *   and changes the {@code updatedAt} timestamp to be the time of update.
      * For the remaining parameters, the existing value is preserved 
      *   if the parameter is null (due to 'keep existing' policy).<br> 
+     * 
      * Preconditions: <br>
      * * {@code newAttributes.getId()} is non-null and
      *  correspond to an existing feedback question. <br>
      */
-    public void updateFeedbackQuestion(FeedbackQuestionAttributes newAttributes) throws InvalidParametersException, EntityDoesNotExistException {
+    public void updateFeedbackQuestion(FeedbackQuestionAttributes newAttributes) throws InvalidParametersException,
+        EntityDoesNotExistException {
```

Comment:
Correct indentation. This one can be indented until it lines up with the other exception.